### PR TITLE
Changes to support a fork of the node-server

### PR DESCRIPTION
**Pull request by MegRiley**
This includes the initial changes I made to support moving from asymmetrik to projecttacoma. I'm guessing I've missed a few scripts but I was able to publish to npm and run the server.

### CODE REVIEW

File: packages/node-fhir-server-core/package.json [hunk 1]
```diff
@@ -37,8 +37,8 @@
     "verbose": true
   },
   "dependencies": {
-    "@asymmetrik/fhir-response-util": "^1.2.4",
-    "@asymmetrik/sof-scope-checker": "^1.0.7",
+    "@projecttacoma/fhir-response-util": "^1.2.4",
+    "@projecttacoma/sof-scope-checker": "^1.0.7",
```
Comment (mgramigna) [inline packages/node-fhir-server-core/package.json:41]: Now that we are only publishing `@projecttacoma/node-fhir-server-core` for now, we can revert these dependencies back to using asymmetrik

File: packages/fhir-sanitize-param/yarn.lock [hunk 2]
```diff
@@ -2,677 +2,19 @@
 # yarn lockfile v1
 
 
-"@babel/code-frame@^7.0.0", "@babel/code-frame@^7.5.5":
```
Comment (mgramigna) [inline packages/fhir-sanitize-param/yarn.lock:5]: I think we can just put this file back. Was probably removed by accident with npm somehow?

**Review by mgramigna — CHANGES_REQUESTED** [review]
Looks good overall, and I see the correct package published to npm. Just some small updates needed noted by the inline comments

[MG1] Also, I see that you added a package-lock here. Asymmetrik uses [yarn](https://yarnpkg.com/) to manage their packages. Usually I prefer npm over yarn, but I think we should just be consistent with what they have. Actions:

* Remove package-lock.json
* (If you don't have yarn) `npm install -g yarn`
* Install dependencies from the root directory of the project with `yarn install`

Commit the deletion to package-lock.json, everything else should be fine

File: packages/fhir-sanitize-param/yarn.lock (hunk 2 above)
Comment (MegRiley) [reply packages/fhir-sanitize-param/yarn.lock:5]: I think it was because I don't remember messing with that file at all

**Review by mgramigna — APPROVED** [review]

**Pull request merged**